Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist_test.py

```diff
@@ -0,0 +1,7 @@
+from FeedGCPWhitelist import Client, google_base_dns
+
+
```

Comment:
Unit tests typically shouldn't make requests. They should be self-contained and testing your logic. Notice how this wouldn't work if the network request failed. Look [here](https://xsoar.pan.dev/docs/integrations/unit-testing) to read about how you should implement your unit-tests.